.github/workflows/ci.yml

```diff
@@ -36,11 +36,21 @@ jobs:
         run: sbt compile Test/compile
 
       - name: Run tests
-        run: sbt test
+        run: |
+          sbt -scala-version 2.12.13 test
+          sbt -scala-version 2.12.14 test
+          sbt -scala-version 2.12.15 test
+          sbt -scala-version 2.13.2 test
+          sbt -scala-version 2.13.3 test
+          sbt -scala-version 2.13.4 test
+          sbt -scala-version 2.13.5 test
+          sbt -scala-version 2.13.6 test
+          sbt -scala-version 2.13.7 test
+          sbt -scala-version 2.13.8 test
```

Comment:
How long does it take now to complete CI?

I would advise splitting these tests into separate job

Reply:
Or maybe add some sort of parallelism within the job (coping whole directory?)

Reply:
It's increased from 6 to 24 minutes. We could have a separate job for every version.

Reply:
> separate job for every version

yeah, probably then the overhead of setting up a new docker container/compilation etc. will outweigh the benefits

Reply:
I created separate jobs for testing. The first job is for testing version 2.12, the second and third is for 2.13. 
Before CI needed ~11 minutes, with sequential testing ~24min, and now ~10 minutes. You can check it and suggest if you have some other preferred way of grouping jobs/tasks.